src/odemis/dataio/tiff.py

```diff
@@ -222,6 +226,13 @@ def _readTiffTag(tfile):
         except (OverflowError, ValueError):
             logging.info("Failed to parse date '%s'", val)
 
+    try:
+        fei_md = tfile.GetField(34682)
+        fei_md = fei_md.decode("latin1") # latin1 -> every char is acceptable
+        md.update(_convertFEItoOdemisMD(fei_md))
+    except Exception as e:
```

Comment:
Can you distinguish between "the tag is not present" (which is going to be 99% of the cases, and which doesn't even require a log message) and "there is FEI metadata but it looked weird", in which case this log is useful.

Reply:
changed to guard (fei_md is None if tag is not present). Will only log if error while parsing / converting metadata